packages/backend/.eslintrc.js

```diff
@@ -9,6 +9,10 @@ module.exports = {
   rules: {
     // Type is enforced by callers. Not entirely, but it's good enough.
     '@typescript-eslint/explicit-function-return-type': 'off',
+    '@typescript-eslint/no-unused-vars': [
+      'error',
+      { ignoreRestSiblings: true },
+    ],
```

Comment:
Isn't it good to have such a rule? Do I understand correctly that this won't complain that `x` is not used, right?
```ts
const { x, y, ...rest } = object
console.log(y, rest)
```

But isn't it good in many cases?